### PR TITLE
[#176068708] getUser for BPD function

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: Ping/function.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (balanza) [inline Ping/function.json]: Why ping? isn't this the same as `info`?

Comment (BurnedMarshal) [reply Ping/function.json]: Ping will be only a ping API, Info instead will perform a healthcheck. For now Info doesn't check the redis connection

Comment (balanza) [reply Ping/function.json]: Ok I see the point.

Then we could have both: 
* `GET /info`, which performs healthcheck and returns 200 with a small payload (service name and version)
* `HEAD /ping` which return 204 and no payload, to minimise the used bandwidth 

Thoughts? cc @gunzip

Comment (gunzip) [reply Ping/function.json]: yup

**Pull request closed without merging**